include/Symmetry.hpp

```diff
@@ -35,12 +31,25 @@ namespace cytnx {
    * --------------|--------------------
    *  Void         |  -99, void type (that means not initialized)
    *  U            |  -1, U1 symmetry
-   *  Z            |  0, Zn symmetry
+   *  Z            |   0, Zn symmetry
+   *  fPar         |  -2, fermionParity symmetry
+   *  fNum         |  -3, fermionNumber symmetry
    *
    *  @see Symmetry::stype(), Symmetry::stype_str()
    */
   extern SymmetryType_class SymType;
 
+  /**
+   * @brief fparity
+   * @details the parity of fermionis
+   * EVEN For bosons or an even number of fermions
+   * ODD  For an even number of fermions
+   */
+  enum fparity : bool { EVEN = false, ODD = true };
```

Comment:
Need to follow the naming rule of enum.

https://google.github.io/styleguide/cppguide.html#Enumerator_Names